kyuubi-server/src/main/scala/org/apache/kyuubi/operation/LaunchEngine.scala

```diff
@@ -37,21 +42,28 @@ class LaunchEngine(session: KyuubiSessionImpl, override val shouldRunAsync: Bool
     setState(OperationState.PENDING)
   }
 
-  override protected def afterRun(): Unit = {
-    OperationLog.removeCurrentOperationLog()
-  }
```

Comment:
what if we keep this?

Reply:
it will close the operation log after runInternal.


Maybe the launch engine operation is still running

Reply:
OperationLog.removeCurrentOperationLog() does not close the log?